GoogleSignIn/Tests/Unit/GIDVerifiedAccountDetailResultTest.m

```diff
@@ -0,0 +1,113 @@
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
+#import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiedAccountDetailResult.h"
+#import "GoogleSignIn/Sources/Public/GoogleSignIn/GIDVerifiableAccountDetail.h"
+
+#import "GoogleSignIn/Sources/GIDVerifyAccountDetail/Fake/GIDVerifiedAccountDetailHandlingFake.h"
+
+#import "GoogleSignIn/Sources/GIDSignIn_Private.h"
+
+#import "GoogleSignIn/Tests/Unit/OIDAuthState+Testing.h"
+#import "GoogleSignIn/Tests/Unit/OIDAuthorizationRequest+Testing.h"
+#import "GoogleSignIn/Tests/Unit/OIDTokenResponse+Testing.h"
+
+@interface GIDVerifiedAccountDetailResultTest : XCTestCase
+@end
+
+@implementation GIDVerifiedAccountDetailResultTest : XCTestCase
+
+- (void)testInit {
+  OIDAuthState *authState = [OIDAuthState testInstance];
+
+  GIDVerifiableAccountDetail *verifiedAccountDetail =
+      [[GIDVerifiableAccountDetail alloc] initWithAccountDetailType:GIDAccountDetailTypeAgeOver18];
+
+  NSArray<GIDVerifiableAccountDetail *> *verifiedList =
+      @[verifiedAccountDetail, verifiedAccountDetail];
+
+  GIDVerifiedAccountDetailResult *result = 
+      [[GIDVerifiedAccountDetailResult alloc] initWithLastTokenResponse:authState.lastTokenResponse
+                                                         accountDetails:verifiedList
+                                                              authState:authState];
+
+  XCTAssertEqual(result.verifiedAuthState, authState);
+  XCTAssertEqual(result.verifiedAccountDetails, verifiedList);
+  XCTAssertEqual(result.expirationDate, authState.lastTokenResponse.accessTokenExpirationDate);
+  XCTAssertEqual(result.accessTokenString, authState.lastTokenResponse.accessToken);
+  XCTAssertEqual(result.refreshTokenString, authState.lastTokenResponse.refreshToken);
+
+}
+
+- (void)testRefreshTokensWithCompletion_success {
+  GIDVerifiableAccountDetail *verifiedAccountDetail = 
+      [[GIDVerifiableAccountDetail alloc] initWithAccountDetailType:GIDAccountDetailTypeAgeOver18];
+
+  NSString *kAccountDetailList = [NSString stringWithFormat:@"%@ %@",
+                                  kAccountDetailTypeAgeOver18Scope,
+                                  kAccountDetailTypeAgeOver18Scope];
+  OIDTokenResponse *tokenResponse = [OIDTokenResponse testInstanceWithScope:kAccountDetailList];
+  OIDAuthState *authState = [OIDAuthState testInstanceWithTokenResponse:tokenResponse];
+  GIDVerifiedAccountDetailHandlingFake *result =
+      [[GIDVerifiedAccountDetailHandlingFake alloc] initWithTokenResponse:authState.lastTokenResponse
+                                                        verifiedAuthState:authState
+                                                                    error:nil];
+
+  NSArray<GIDVerifiableAccountDetail *> *expectedVerifiedList =
+      @[verifiedAccountDetail, verifiedAccountDetail];
+
+  XCTestExpectation *expectation = [self expectationWithDescription:@"Completion called"];
```

Comment:
"Completion called" is not informative. Update this to more specifically describe the case; something like, "Refreshed verified account details completion called."

Reply:
Done.